tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/ProgrammeMembershipResource.java

```diff
@@ -160,6 +160,52 @@ public ResponseEntity<ProgrammeMembershipDTO> getProgrammeMembership(@PathVariab
     return ResponseUtil.wrapOrNotFound(Optional.ofNullable(programmeMembershipDto));
   }
 
+  /**
+   * GET /programme-memberships/{ids}/summary-list : get the list of programmeMembership summaries.
+   *
+   * @param ids a comma-separated list of programmeMembershipDTO ids to retrieve
+   * @return the ResponseEntity with status 200 (OK) and with body the programmeMembershipDTO list,
+   *         or with status 404 (Not Found)
+   */
+  @GetMapping("/programme-memberships/{ids}/summary-list")
+  @PreAuthorize("hasPermission('tis:people::person:', 'View')")
+  public ResponseEntity<List<ProgrammeMembershipSummaryDto>> getProgrammeMembershipSummaryList(
+      @PathVariable String ids) {
+
+    String[] idArray = ids.split(",");
+    List<ProgrammeMembershipSummaryDto> summaryList = new ArrayList<>();
+
+    for (String id : idArray) {
+      ProgrammeMembershipDTO programmeMembershipDto = null;
+
+      try {
+        programmeMembershipDto = programmeMembershipService.findOne(UUID.fromString(id));
```

Comment:
I forgot to say, I think there's probably a more appropriate method on the service to use for this search rather than looping over each id separately to do a `findOne`.